scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -219,6 +226,29 @@ workflow CNVSomaticPairWorkflow {
             gatk_docker = gatk_docker,
             disk_space_gb = plot_normal_disk
     }
+
+    if (is_run_oncotator) {
+        call Oncotate.CNVOncotateCalledSegments as OncotateCalledCNVWorkflow {
+            input:
+                 called_file = CallCopyRatioSegmentsTumor.called_copy_ratio_segments,
+                 oncotator_docker = oncotator_docker
+        }
+    }
+
+    output {
+        File modeled_segments_tumor = ModelSegmentsTumor.modeled_segments
+        File called_copy_ratio_segments_tumor = CallCopyRatioSegmentsTumor.called_copy_ratio_segments
+        File denoised_read_counts_tumor = DenoiseReadCountsTumor.denoised_copy_ratios
+        File standardized_copy_ratios_tumor = DenoiseReadCountsTumor.standardized_copy_ratios
+
+        File modeled_segments_normal = ModelSegmentsNormal.modeled_segments
+        File called_copy_ratio_segments_normal = CallCopyRatioSegmentsNormal.called_copy_ratio_segments
+        File denoised_read_counts_normal = DenoiseReadCountsNormal.denoised_copy_ratios
+        File standardized_copy_ratios_normal = DenoiseReadCountsNormal.standardized_copy_ratios
+
+        File oncotated_called_file_tumor = select_first([OncotateCalledCNVWorkflow.oncotated_called_file, "null"])
+        File oncotated_called_gene_list_file_tumor = select_first([OncotateCalledCNVWorkflow.oncotated_called_gene_list_file, "null"])
+    }
```

Comment:
Why not expose more output files?  Otherwise, why run e.g. the plotting tasks at all?

I'd just go ahead and expose every single output.

Reply:
Done